seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -2967,7 +2967,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*\\s+style\\s*=\\s*(?:\\'|\")\\s*(?:[\\s\\S]*;+)*\\s*color\\s*:\\s*white\\s*(?:;+[\\s\\S]*;*)*(?:\\'|\")+/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*\\s+style\\s*=\\s*(?:\\'|\")\\s*(?:[\\s\\S]*;+)*\\s*color\\s*:\\s*white\\s*(?:;+[\\s\\S]*;*)*(?:\\1)+/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
@hay-dee Sorry for not responding sooner!

I think the easiest way would be to 
1. make the changes on your local branch
2. `git commit --amend` the changes into your previous commit. This keeps a single commit and helps making the commit history of fCC cleaner.
3. `git push -f <your-fork-on-github, most likely "origin"> <your-branch>`. E.g: `git push -f origin fix/validate-inline-style-color-updated`. The `-f` flag means "force" and tells GitHub to overwrite your previous changes, which allows you to change history by using `git commit --amend`.

Feel free to talk to other contributors in the [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) if you need further help! They may answer faster than me :smile:

Thanks for your help!